src/main/java/menion/android/whereyougo/maps/utils/LocusMapDataProvider.java

```diff
@@ -28,23 +35,16 @@
 import cz.matejcik.openwig.EventTable;
 import cz.matejcik.openwig.Zone;
 import cz.matejcik.openwig.formats.CartridgeFile;
-import locus.api.android.objects.PackWaypoints;
-import locus.api.objects.extra.ExtraData;
-import locus.api.objects.extra.ExtraStyle;
-import locus.api.objects.extra.ExtraStyle.LineStyle.ColorStyle;
-import locus.api.objects.extra.ExtraStyle.LineStyle.Units;
 import locus.api.objects.extra.Location;
-import locus.api.objects.extra.Track;
-import locus.api.objects.extra.Waypoint;
 
```

Comment:
Please reorder the imports.
Also in the other files, I would say.

Reply:
Done in 2daea5a655714047859cba8bb48e8b816def4518. Surprisingly, Idea can't do this by default. You need to install and activate the plugin.